doc/flame/components.md

```diff
@@ -158,6 +158,24 @@ available eventually: after they are loaded and mounted. We can only assure
 that they will appear in the children list in the same order as they were
 scheduled for addition.
 
+#### Ensuring a component has a given parent
```

Comment:
I did that as a fourth as I didnt find a good place to put this, and this chapter talks about children order and parent relations so I thought this might be a nice fit.

Reply:
I think this would be fine as a third level header too, if you look at the section after this one it is also third level and I think they fit on the same level.